scripts/app.js

```diff
@@ -23,10 +33,30 @@ const formElement = document.getElementById("config-form");
 
 const playerValidationParagraph = document.getElementById("config-errors");
 
+//active game variables
+const startGameButton = document.getElementById("start-game");
+const activeGameSection = document.getElementById("game-started");
+const playerNameAlertModal = document.getElementById("player-name-alert");
+
+const playerTurnElement = document.getElementById("active-player-name-element");
+
+const gameBoardElements = document.querySelectorAll("#game-board-element li");
+
+const gameOverArticle = document.getElementById("game-over-element");
+const gameOverHeading = document.getElementById("game-over-heading");
+
+//config event listeners
```

Comment:
Do you want to create another branch and have a go at adding all these event listeners directly in the html?